docker-hub/repos.md

```diff
@@ -47,9 +47,9 @@ images to this repository.
 
 2. Select a repository from the list, click **Settings** and then Delete Repository.
 
-  > **Note:**
-  >
-  > Deleting a repository deletes all the images it contains and its build settings. This action cannot be undone.
+    > **Note:**
+    >
+    > Deleting a repository deletes all the images it contains and its build settings. This action cannot be undone.
```

Comment:
Slightly leaning towards making this "note" an "important" / "warning", and putting it at the top, to make the numbered list more to-the-point. i.e. something like

```markdown
## Deleting a repository

> **Important**
>
> Deleting a repository deletes all the images it contains and its build settings. This action cannot be undone.
{: .important }


1. Sign into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and click **Repositories**.
2. Select a repository from the list, click **Settings** and then Delete Repository.
3. Enter the name of the repository to confirm the deletion and click **Delete**.
```


## Deleting a repository

> **Important**
>
> Deleting a repository deletes all the images it contains and its build settings. This action cannot be undone.
{: .important }


1. Sign into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and click **Repositories**.
2. Select a repository from the list, click **Settings** and then Delete Repository.
3. Enter the name of the repository to confirm the deletion and click **Delete**.

Reply:
Actually, given that the actual UX already shows a confirmation, perhaps an `info` is good enough, as it's more a "FYI"

(in general, we should reduce the number of "info", "warning" blocks, as sometimes they're basically just part of the description 😅 )

Reply:
Could we circle back round to this one as a follow-up? I'd like to keep this stack of patches just minimal linting changes is possible, without any major changes in content.

Reply:
Yup, works for me!